src/Readarr.Api.V1/Languages/LanguageResource.cs

```diff
@@ -0,0 +1,13 @@
+using Newtonsoft.Json;
```

Comment:
This should probably be `System.Text.Json` - the api doesn't use Newtonsoft to serialize